events/README.md

```diff
@@ -0,0 +1,416 @@
+    "user_id": "<user_id>",
+    "role": "<role>"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "token": "<auth_token>",
+        "msg": "Token generated successfully!",
+        "success": true
+      }
+      ```
+- **Error Response:**
+  - **Code:** 500
+    - **Content**
+      **`{ "msg": "Some error occured!", "success": false }`**
+
+## GET - /rooms/:id=<room_id>
+
+Retrieves the details of a specific active room.
+
+- **Params:**
+  - **`id=[string]`** (The ID of the room to retrieve)
+- **Query**
+  - None
+- **Body**
+  - None
+- **Headers**
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<room_id>",
+        "name": "<room_name>",
+        "customer_id": "<customer_id>",
+        "session": {
+          "id": "<session_id>",
+          "created_at": "",
+          "peers": ["peer_id", "peer_id", "peer_id"]
+        }
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      **`{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`**
+  - **Code:** 404
+    - **Content**
+      **`{ 'statusCode': 404, 'error': 'Not Found', 'message': 'Room not found' }`**
+  - **Code:** 500
+    - **Content**
+      **`{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Unable to retrieve room details' }`**
+
+## PUT - /rooms
+
+Update the room, make room enabled/disabled,
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - Required: `id=[string]` (room id)
+  - Required: `enabled=[boolean]` (Flag to indicate if the room is enabled)
+  ```json
+  {
+    "id": "<room_id>",
+    "enabled": false
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "message": "Room is enabled"
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't update room. Please try again later'`
+
+## DELETE - /rooms
+
+Trigger this request to end an active room.
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - **`<room_id>`** (required) : string - The ID of the room to end an active room.
+  - Required: **`reason=[string]`** (reason for ending the room)
+  - Optional: **`lock=[boolean]`** (if true, no new peers will be allowed to join the room after it is ended.)
+  ```json
+  {
+  		"id": "<room_id>"
+      "reason": "Class has ended",
+      "lock": false
+  }
```

Comment:
the formatting for `id` seems to be out of place

Reply:
Fixed it.